docs/artifacts-storage.md

```diff
@@ -0,0 +1,48 @@
+# Artifacts storage
+
+Testkube supports test artifacts collection. 
+
+Currently only Cypress executor job is producing test artifacts. Executor will scrape the files and store them in [Minio](https://min.io/).  The executor will create a bucket named by execution ID and collect all files that are stored in Cypress artifacts location `Cypress/`
+
+Available configuration parameters in helm charts:
+
+
+| Parameter                          | Is optional | Default                              | Default                                              |
+| ---------------------------------- | ----------- | ------------------------------------ | ---------------------------------------------------- |
+| api-server.storage.endpoint        | yes         | testkube-minio-service-testkube:9000 | URL of the S3 bucket                                 |
+| api-server.storage.accessKeyId     | yes         | minio                                | Access Key ID                                        |
+| api-server.storage.accessKey       | yes         | minio123                             | Access Key                                           |
+| api-server.storage.location        | yes         |                                      | Region                                               |
+| api-server.storage.token           | yes         |                                      | S3 Token                                             |
+| api-server.storage.SSL             | yes         | false                                | Indicates whether SSL communication is to be enabled |
+| api-server.storage.scrapperEnabled | yes         | true                                 | Indicates whether executors should scrape artifacst  |
+
+
+The API Server is accepting following environment variables:
```

Comment:
Are these environment variables mandatory?

If so: `The API Server expects following environment variables:`

Or if not `The API Server accepts following environment variables:`

Reply:
```suggestion
The API Server accepts the following environment variables:
```